Testing/Exec/ReactEval_C/inputs.3d-regt

```diff
@@ -26,11 +26,13 @@ cvode.solve_type = dense
 # Max size of problem
 max_grid_size  = 16
 # Choose name of output pltfile
-amr.plot_file  = plt
+amr.plot_file  = /gpfs/wolf/gen138/scratch/balos1/hackathon/plt
```

Comment:
Why is this here?

Reply:
Was an old commit, this has since been changed (and the commit squashed).